lab1_task2.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE HTML>
```

Comment:
Watch the indentation:

```html
<body>

 <h1> Просто об HTTP</h1>
 <ul>
    <li>Как отправить HTTP-запрос?</li>
    <li>Как прочитать ответ?</li>
    <li>А что с безопасностью?</li>

```

Reply:
Nice use of classes, `dfn`, `code`, and `dfn` tags and HTML entities.

Reply:
thanks. I did the validation. seems all fixed now